explore/src/main/scala/explore/HelpBody.scala

```diff
@@ -75,7 +76,7 @@ object HelpBody {
         load(props.url).flatMap(v => state.set(Pot.fromTry(v)).to[IO])
       }
       .render { (props, state) =>
-        val imageConv = (s: Uri) => props.baseUrl.withPath(s.path)
+        val imageConv = (s: Uri) => props.baseUrl.withPath(props.baseUrl.path.concat(s.path))
```

Comment:
Seems weird that you need to do it this way. But, if it works, it works. :)

Reply:
yeah, it is a bit weird